python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -251,17 +263,18 @@ def assert_any_corrections(self, data):
         AssertionError
             If one of more attribute of the topic is set to True.
         """
-        assert any([getattr(data, corr)
-                    for corr in self.valid_corrections]), \
+
+        assert any([getattr(data, corr, False)
+                    for corr in self.corrections]), \
             "At least one correction must be set."
 
     def assert_corrections(self, mode):
         """Check that the corrections are either enabled or disabled.
 
         Parameters
         ----------
-        mode : str
-            Either enabled or disabled
+        mode : int
+            Specify if a correction is disabled (0) or enabled (1).
```

Comment:
Thank you for changing this from a str. Do you anticipate more than 2 modes? If not then a bool is better because it can only *have* 2 values (no need to check the value passed in for validity). But you'd have to rename it, e.g. to `enabled`